src/internal/operators/timestamp.ts

```diff
@@ -8,24 +8,23 @@ import { map } from './map';
  * The `timestamp` operator maps the *source* observable stream to an object of type
  * `{value: T, timestamp: R}`. The properties are generically typed. The `value` property contains the value
  * and type of the *source* observable. The `timestamp` is generated by the schedulers `now` function. By
- * default it uses the *async* scheduler which simply returns `Date.now()` (milliseconds since 1970/01/01
+ * default, it uses the `asyncScheduler` which simply returns `Date.now()` (milliseconds since 1970/01/01
```

Comment:
I have a fix [for this](https://github.com/ReactiveX/rxjs/pull/6247#issuecomment-825491834) in my [other branch](https://github.com/jakovljevic-mladen/rxjs/tree/remove_unnecessary_images). If changes from that branch land to master, `asyncScheduler` will automatically become a link to the `asyncScheduler` docs.